Dockerfile

```diff
@@ -0,0 +1,82 @@
+# syntax = docker/dockerfile:1
+
+# This Dockerfile is designed for production, not development. Use with Kamal or build'n'run by hand:
+# docker build -t my-app .
+# docker run -d -p 80:80 -p 443:443 --name my-app -e RAILS_MASTER_KEY=<value from config/master.key> my-app
+
+# Make sure RUBY_VERSION matches the Ruby version in .ruby-version
+ARG RUBY_VERSION=3.3.5
```

Comment:
The image is at `3.3.5` while the application in this MR is still at `3.3.4`. I totally understand why (so many PRs), but it conflicts with the comment above. ;)

Reply:
Yes. I've bumped this commit to 3.3.5 already, because it would have been overlooked easily in https://github.com/openHPI/codeocean/pull/2517. Since https://github.com/openHPI/codeocean/pull/2517 is merged first, the commit history on `master` will be clean.